package.json

```diff
@@ -8,8 +8,7 @@
     "build": "ng build",
     "test": "ng test",
     "lint": "ng lint",
-    "e2e": "ng e2e",
-    "deploy": "ng build --prod && firebase deploy"
```

Comment:
I suggest that you keep the deploy script since we have npm scripts for all the other tasks already:

`"deploy": "ng deploy"`